components/Socials/sub-components/SocialType.jsx

```diff
@@ -0,0 +1,137 @@
+            />
+          </g>
+        </StyledSvg>
+      )}
+
+      {type === 'youtube' && (
+        <StyledSvg
+          role="img"
+          aria-labelledby="youtube-title"
+          xmlns="http://www.w3.org/2000/svg"
+          width={size}
+          height={size}
+          viewBox={`0 0 ${SVGOriginalSizeWithBorder} ${SVGOriginalSizeWithBorder}`}
+        >
+          <title id="youtube-title">{description}</title>
+          <g id="Box_Youtube" transform="translate(350 -35)">
+            <rect
+              id="Rectangle_3"
+              data-name="Rectangle 3"
+              width={SVGOriginalSize}
+              height={SVGOriginalSize}
+              rx="8"
+              transform="translate(-349 36)"
+              fill="#fff"
+              stroke="#424242"
+              strokeWidth="2"
+            />
+            <path
+              id="Path_20"
+              data-name="Path 20"
+              d="M422.05,96.9a3.711,3.711,0,0,0-2.6-2.617c-2.283-.617-11.45-.617-11.45-.617s-9.167,0-11.467.617a3.685,3.685,0,0,0-2.6,2.617,39.409,39.409,0,0,0-.6,7.1,38.348,38.348,0,0,0,.617,7.1,3.711,3.711,0,0,0,2.6,2.617c2.283.616,11.45.616,11.45.616s9.167,0,11.467-.616a3.687,3.687,0,0,0,2.6-2.617,39.409,39.409,0,0,0,.6-7.1A38.348,38.348,0,0,0,422.05,96.9Z"
+              transform="translate(-734 -45)"
+              fill="#ee3124"
+            />
+            <path
+              id="Path_21"
+              data-name="Path 21"
+              d="M405,108.367V99.633L412.667,104Z"
+              transform="translate(-734 -45)"
+              fill="#fff"
+            />
+          </g>
+        </StyledSvg>
+      )}
+
+      {type === 'facebook' && (
+        <StyledSvg
+          role="img"
+          aria-labelledby="facebook-title"
+          xmlns="http://www.w3.org/2000/svg"
+          width={size}
+          height={size}
+          viewBox={`0 0 ${SVGOriginalSize} ${SVGOriginalSize}`}
+        >
+          <title id="facebook-title">{description}</title>
+          <g id="Box_Facebook" transform="translate(606 -35)">
+            <rect
+              id="Rectangle_1"
+              data-name="Rectangle 1"
+              width={SVGOriginalSize}
+              height={SVGOriginalSize}
+              rx="9"
+              transform="translate(-606 35)"
+              fill="#3c5193"
+            />
+            <path
+              id="Path_4"
+              data-name="Path 4"
+              d="M146.667,100.389h2.285V98.246a5.551,5.551,0,0,1,.737-3.306,4.115,4.115,0,0,1,3.547-1.606,14.874,14.874,0,0,1,4.1.4L156.763,97a7.945,7.945,0,0,0-1.842-.267c-.888,0-1.685.309-1.685,1.165v2.493h3.645l-.254,3.192h-3.391v11.086h-4.284V103.581h-2.285Z"
+              transform="translate(-734 -45)"
+              fill="#fff"
+              fillRule="evenodd"
+            />
+          </g>
+        </StyledSvg>
+      )}
+    </SocialWrapper>
+  );
+};
+
+SocialType.defaultProps = {
+  size: '48',
+  description: 'Social description',
```

Comment:
This should be either required or empty. If user "forgets" to pass this prop description would be too generical.